backend/src/main/kotlin/com/group7/artshare/service/ProfileService.kt

```diff
@@ -28,4 +35,47 @@ class ProfileService(private val registeredUserService: RegisteredUserService, p
             }
         }
     }
+
+
+    fun getSettings(user: RegisteredUser) : SettingDTO {
+        var settings  = SettingDTO()
+        settings.email = user.getEmail()
+        settings.username = user.username
+        settings.name = user.accountInfo?.name
+        settings.surname = user.accountInfo?.surname
+        settings.country = user.accountInfo?.country
+        settings.dateOfBirth = user.accountInfo?.dateOfBirth
+        settings.profilePictureId = user.accountInfo?.profilePictureId
```

Comment:
We should check whether the corresponding image id is in the database or not. If it does not exist, we should throw an error. You can examine the ArtItem and Event endpoints similar [structure](https://github.com/bounswe/bounswe2022group7/pull/434)